app/webpacker/components/Tickets/index.jsx

```diff
@@ -15,7 +15,7 @@ import updateStatus from './api/updateStatus';
 export default function Wrapper({ id }) {
   return (
     <WCAQueryClientProvider>
-      <Tickets id={id} />
+      <Tickets id={parseInt(id, 10)} />
```

Comment:
This feels a little bit wrong. If you are so certain that it can definitely always be parsed as an integer, then why not just pass in an integer from the outside world in the first place?

Or in other words, under what circumstance is the `id` prop in `Wrapper` **not** an integer, and could you just fix that (instead of manually parsing here)?

Reply:
Okay actually I was under the assumption that in future for next, the ID from URL will always be string. Now I did research, and we can actually make it int by defining type for the prop. So sending it as integer from backend makes sense. Changed.